Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +47,23 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array != null && array.Length != 0)
+            {
+                var arrayLength = array.Length;
+                // make a copy of array by value (not reference)
+                int[] tmp = new int[128];
+                for (var i = 0; i < arrayLength; i++)
+                {
+                    tmp[i] = array[i];
+                }
+
+                // reverse current index with last index reduced with current index
+                for (var i = 0; i < arrayLength; i++)
+                {
+                    //Console.WriteLine("Swapping: " + array[i] + " and " + tmp[arrayLength - i-1]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011611)